### DISCUSSION TITLE
Strip diffcalc errors in logs

### DISCUSSION THREAD

**Discussion by PercyDan54** [ppy/osu#16243] [Ideas] [answered]
I got this while diffcalc some aspire maps from stable
Maybe change it to: Failed to process line xxx on file xxx
![image](https://user-images.githubusercontent.com/50285552/147385523-92fa0650-85b2-4e64-861c-26f64023c498.png)

**Comment by peppy** [comment, marked as answer]
I'm not sure what the issue is. Broken input, broken output. Log is fine.

**Reply by PercyDan54** [reply]
The log is way too long
